Gemfile

```diff
@@ -1,14 +1,5 @@
 source "https://rubygems.org"
 
-# Declare your gem's dependencies in event_capture.gemspec.
-# Bundler will treat runtime dependencies like base dependencies, and
-# development dependencies will be added by default to the :development group.
 gemspec
 
-# Declare any dependencies that are still in development here instead of in
-# your gemspec. These might include edge Rails or gems from your path or
-# Git. Remember to move these dependencies to your gemspec before releasing
-# your gem to rubygems.org.
-
-# To use debugger
-# gem 'debugger'
+gem "activerecord", "~> 4.1"
```

Comment:
No, it is for Rails 4.1 and 4.2. There is a separate gemfile under `gemfiles/` used by Travis.